lib/agent/socket_util.js

```diff
@@ -72,7 +72,7 @@ exports.secureSocket = function (socket, host, agent, mock)
   const validate = function ()
   {
     // stop listening for the secure event
-    socket.removeListener('secure', validate);
+    socket.removeListener('secureConnect', validate);
```

Comment:
let's create a const for this event name

Reply:
Done.